openshift-e2e.yaml

```diff
@@ -0,0 +1,23 @@
+keda:
+  skipSetup: true
+  skipCleanup: true
```

Comment:
Super cool! Is the config handling such that potentially later we would want to replace our setup/cleanup direct calls with test runs against minimal config like: 
```
keda:
  skipSetup: true
  skipCleanup: false
```
 Where our `e2e-test-openshift-clean` makefile target would end up like this someday: 
 ```
E2E_TEST_CONFIG=openshift-e2e-clean.yaml go run -tags e2e ./tests/run-all.go
```
Or nah?

Reply:
I don't think the code is set up in a way to do that. I didn't allow the `run-all.go` filtering to run 0 tests on purpose because I assumed that meant there is some human error with specifying which tests you wanted to run.

That is an interesting thought though, but I was only thinking of this configuration from a perspective where you would only have to run the `run-all.go` file once, (and since we already had relatively simple hacks for this, keep the separate clean + setup make targets just simple `go run` calls.